src/components/organisms/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,98 @@
+import DesktopNavigation from "@components/organisms/Navbar/DesktopNavigation";
+import { useEffect, useState } from "react";
+import NavbarLogo from "./components/logo";
+import Profile from "./components/profile";
+import MobileDropdown from "./MobileDropdown";
+import { styled } from "@stitches/react";
+import { createStyles } from "@theme";
+
+const Navbar = () => {
+  const [color, setColor] = useState("transparent");
+  const [shadow, setShadow] = useState("0 0.1px 0.2px #0d0d0d");
+
+  const changeNavbarColor = () => setColor("#fff");
+  const revertNavbarColor = () => setColor("transparent");
+
+  const changeShadow = () => setShadow("0 0.1px 1px #0d0d0d");
+  const revertShadow = () => setShadow("0 0.1px 0.2px #0d0d0d");
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (window.scrollY > 40) {
+        changeNavbarColor();
+        changeShadow();
+      }
+      if (window.scrollY < 40) {
+        revertNavbarColor();
+        revertShadow();
+      }
+    };
+
+    handleScroll();
+
+    window.addEventListener("scroll", handleScroll);
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
+
+  return (
+    <Container css={{ boxShadow: shadow, backgroundColor: color }}>
+      {/*TODO: Split into Mobile Navbar and Desktop Navbar components and use ternary operator*/}
```

Comment:
do this todo

Reply:
probably pass in the tab items mentioned above so that mobile and desktop can share them somehow